components/src/core/components/Input/RadioPills/RadioPillGroup.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <div
+    :id="id"
+    class="radio-pill-group oxd-input-field-bottom-space"
+    :style="style"
+  >
+    <oxd-radio-pill
+      v-for="(option, index) in options"
+      :key="option.id"
+      :id="id + '_' + option.id"
+      :value="option.id"
+      :style="option.style"
+      :autofocus="$attrs.autofocus && index === 0"
+      :name="$attrs.name"
```

Comment:
might have to make name a required prop since according to html sepc, for radio group to work all radios must have same name. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio#defining_a_radio_group

Reply:
Added name as a required prop.